core/kits/assets/js/panel-menu.js

```diff
@@ -13,18 +13,31 @@ PanelMenu.createGroupItems = ( groupName, keys ) => {
 
 	return keys.map( ( key ) => {
 		const fullKey = groupName + '-' + key,
-			tab = tabs[ fullKey ];
+			tab = tabs[ fullKey ],
+			item = {
+				name: fullKey,
+				icon: tab.icon,
+				title: tab.title,
+			};
 
-		return {
-			name: fullKey,
-			icon: tab.icon,
-			title: tab.title,
-			callback: () => $e.route( 'panel/global/' + fullKey ),
-		};
+		if ( ! 'additional-settings' === key ) {
```

Comment:
Remove the 'additional-settings' key from the items array, and you won't need to handle it here